apps/portals/admin/src/screens/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Box,
+  CategoryCard,
+  Divider,
+  GridColumn,
+  GridContainer,
+  GridRow,
+  Icon,
+  useBreakpoint,
+  Text,
+} from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { Link } from 'react-router-dom'
+import { m, PortalNavigationItem, useNavigation } from '@island.is/portals/core'
+import { m as delegationsMessages } from '@island.is/portals/shared-modules/delegations'
+import partition from 'lodash/partition'
+import * as styles from './Dashboard.css'
+
+export const Dashboard = () => {
+  const { formatMessage } = useLocale()
+  const navigation = useNavigation()
+  const { md } = useBreakpoint()
+
+  const [topNavigation, bottomNavigation] = partition(
+    navigation?.children || [],
+    (navItem) =>
+      formatMessage(navItem.name) !==
+      formatMessage(delegationsMessages.accessControl),
```

Comment:
It would be nice to have a better way to partition these rather than comparing translated text.

One is to have separate navigations for top and bottom.

```
const top = useNavigation(TOP_NAVIGATION)
const bottom = useNavigation(BOTTOM_NAVIGATION)
```

Reply:
Yes I thought about that, but I think in the end the `id` field on at least the navigation item would be great and also on the portal module.

The way you are suggesting would mean that we would have to either manually provide the top and bottom navigation or if you are thinking as a constants we would still have to filter out the desired navItems.

What about optional id field and then when service-portal moves over to the new portal structure, then we can make it required.

Reply:
Extracted into `useAdminNavigation()` hook and added required id field to all modules and optional id field to PortalNavigationItem.

Reply:
Nevermind the above went you way :)